src/platform/site-wide/user-nav/containers/Main.jsx

```diff
@@ -5,16 +5,19 @@ import URLSearchParams from 'url-search-params';
 
 import { isInProgressPath } from 'platform/forms/helpers';
 import FormSignInModal from 'platform/forms/save-in-progress/FormSignInModal';
-import { SAVE_STATUSES } from 'platform/forms/save-in-progress/actions';
-import { getBackendStatuses } from 'platform/monitoring/external-services/actions';
-import { updateLoggedInStatus } from 'platform/user/authentication/actions';
 import SessionTimeoutModal from 'platform/user/authentication/components/SessionTimeoutModal';
 import SignInModal from 'platform/user/authentication/components/SignInModal';
 import { initializeProfile } from 'platform/user/profile/actions';
-import { hasSession } from 'platform/user/profile/utilities';
-import { isLoggedIn, isProfileLoading, isLOA3 } from 'platform/user/selectors';
 import environment from 'platform/utilities/environment';
+import { hasSession, hasSessionSSO } from 'platform/user/profile/utilities';
```

Comment:
regrouping just for more clarity

Reply:
ahhh, I see. I didn't notice that before. I was attempting to group by type (selectors, actions, components, etc). Just my way of making sense of everything going on